vmm/src/memory_manager.rs

```diff
@@ -56,6 +56,9 @@ const SNAPSHOT_FILENAME: &str = "memory-ranges";
 #[cfg(target_arch = "x86_64")]
 const X86_64_IRQ_BASE: u32 = 5;
 
+#[cfg(target_arch = "x86_64")]
+const SGX_PAGE_SIZE_SHIFT: u64 = 12;
```

Comment:
You could go one step further and declare another constant:
```rust
#[cfg(target_arch = "x86_64")]
const SGX_PAGE_SIZE: u64 = 1 << SGX_PAGE_SIZE_SHIFT;
```

Reply:
sure